tox.ini

```diff
@@ -29,12 +29,21 @@ description =
     alldeps: with all optional dependencies
     devdeps: with the latest developer version of key dependencies
     oldestdeps: with the oldest supported version of key dependencies
-    cov: and test coverage
 setenv =
     MPLBACKEND = agg
     COLUMNS = 180
-    PYTEST_COMMAND = pytest -vvv --pyargs sunpy --cov=sunpy --cov-config={toxinidir}/setup.cfg {toxinidir}/docs
+    PYTEST_COMMAND = pytest -vvv --pyargs sunpy --cov-report= --cov=sunpy --cov-config={toxinidir}/setup.cfg {toxinidir}/docs
     astropydev,build_docs,online: HOME = {envtmpdir}
+    SUNPY_SAMPLEDIR = {toxinidir}/.tox/{envname}/sample_data/
+passenv =
+    CURL_CA_BUNDLE
```

Comment:
I stole this from the tox/tox/tox.ini for fun.

Reply:
Add a comment as to why?

Reply:
I dont know.